forc-test/src/lib.rs

```diff
@@ -197,13 +190,13 @@ impl<'a> PackageTests {
         let pkg_with_tests = self.built_pkg_with_tests();
         // TODO: We can easily parallelise this, but let's wait until testing is stable first.
         let tests = pkg_with_tests
-            .entries
+            .pkg_entries
             .iter()
             .filter(|entry| entry.is_test())
             .map(|entry| {
```

Comment:
Another option could be something like:

```suggestion
            .filter_map(|entry| entry.kind.test().map(|test| (entry, test)))
            .map(|(entry, test_entry)| {
```
This might allow us to remove the second unnecessary `entry.expect_test_entry()` check below